students/HarveyHerela/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,16 @@
+for i in range(1,101):
+    fizz, buzz = False, False
+    if (i % 3) == 0:
+        fizz = True
+    if (i % 5) == 0:
+        buzz = True
+    
+    if fizz or buzz:
+        if fizz:
+            print("Fizz", end = '')
+        if buzz:
+            print("Buzz", end = '')
+        print()
+    else:
+        print(i)

```

Comment:
this certainly works! but can you think of another way that is more inline and less code?

Reply:
Yes. I could have just 1 Boolean instead of 2, and it would only keep track if fizz/buzz/fizzbuzz was printed instead of determining what to print. Should I update my code and do another pull request?

Reply:
You can do it with no booleans at all 😉 

not required, but if you have time I would say give it another try (yes, it is truly given at interviews)!